pylint/test/functional/consider_merging_isinstance.txt

```diff
@@ -0,0 +1,5 @@
+consider-merging-isinstance:15:isinstances:Consider merging these isinstance calls to var[3].
```

Comment:
This is a good approach.
This require a full refactoring of the current methods.
Are you agree if I create a new issue with this request and I work it in other PR?